core/controllers/questions_list.py

```diff
@@ -89,3 +89,29 @@ def get(self, comma_separated_skill_ids):
             'next_start_cursor': next_start_cursor
         })
         self.render_json(self.values)
+
+
+class QuestionCountDataHandler(base.BaseHandler):
+    """Provides data regarding the number of questions assigned to the given
+    skill ids.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self, comma_separated_skill_ids):
+        """Handles GET requests."""
+        skill_ids = comma_separated_skill_ids.split(',')
+
+        for skill_id in skill_ids:
```

Comment:
This looks like a duplicate code from line 42-46 should we move this to a function and use it in both the places?

Reply:
Done.